src/hooks/useProcessReleases.ts

```diff
@@ -115,22 +104,21 @@ function useProcessReleases(
 ): UseProcessReleasesReturn {
 	const [processedReleases, setProcessedReleases] =
 		useState<ProcessedReleasesCollection | null>(null)
-	const [isProcessing, setIsProcessing] = useState(false)
+	const [isProcessing, setIsProcessing] = useState(true)
```

Comment:
This was another problem: the initial state must be true, so it starts as loading. That was the actual cause of the error blink in the results page.

I've reworked the hook so it's easier to follow.